README.md

```diff
@@ -165,11 +169,11 @@ The BigQuery to VCF pipeline is used to export variants in BigQuery to one VCF f
 Please refer to [BigQuery to VCF pipeline](docs/bigquery_to_vcf.md) for more
 details.
 
-## Running jobs in a particular region/zone
+## Running jobs in a particular region
 
-You may need to constrain Cloud Dataflow job processing to a specific geographic
-region in support of your project’s security and compliance needs. See
-[Setting zone/region doc](docs/setting_zone_region.md).
+You must constrain Cloud Dataflow job processing to a specific geographic
```

Comment:
How about

You must choose a geographic region for Cloud Dataflow to process your data by setting ...

Reply:
done.